src/main/java/uk/gov/hmcts/darts/common/entity/Case.java

```diff
@@ -5,76 +5,64 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
-import jakarta.persistence.JoinColumn;
-import jakarta.persistence.JoinTable;
-import jakarta.persistence.ManyToMany;
-import jakarta.persistence.ManyToOne;
 import jakarta.persistence.OneToMany;
+import jakarta.persistence.SequenceGenerator;
 import jakarta.persistence.Table;
 import jakarta.persistence.Version;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
+import lombok.Data;
 
 import java.time.OffsetDateTime;
+import java.util.ArrayList;
 import java.util.HashSet;
-import java.util.Objects;
+import java.util.List;
 import java.util.Set;
 
 @Entity
 @Table(name = "moj_case")
-@NoArgsConstructor
-@Getter
-@Setter
+@Data
 @SuppressWarnings({"PMD.ShortClassName"})
 public class Case {
 
     private static final String MAPPED_BY_THE_CASE = "theCase";
 
     @Id
     @Column(name = "moj_cas_id")
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "moj_cas_gen")
+    @SequenceGenerator(name = "moj_cas_gen", sequenceName = "moj_cas_seq", allocationSize = 1)
```

Comment:
i don't think we should have an allocationSize of 1

Reply:
Need to leave at 1 for now to match our sql. Barry is going to review the proposed schema and will update his scripts with some sequences having a modified `increment` and `cache` options and we can change our entities to match after that.